### PR TITLE
Desktop Feature Implementation

### PR DESCRIPTION
**Pull request by denscholar**
I successfully implemented the desktop feature of the application; including all styles and images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -170,30 +176,11 @@ <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
...
-              </div>
-            </div>
-          </div>
-        </div>
       </div>
     </div>
   </section>
   <!-- About Myself -->
-  <section class="about-me">
+  <section class="about-me" id="about">
```
Comment (petudeveloper) [inline index.html]: - [ ] Your design is fantastic. In order for your design to be perfect please fix the horizontal scrolling you are presenting, it seems that one of your sections is not adapting to its container.  💪 💪 💪 
Please try adding the corresponding CSS property to stick to the design as much as possible.

Check the image below to have a better understanding.
Capture

**Review by petudeveloper — CHANGES_REQUESTED** [review]
Hi @denscholar , 

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -211,44 +197,44 @@ <h1>About me</h1>
...
-          <li>CodeKit</li>
-          <li>GitHub</li>
-          <li>Codepen</li>
-          <li>GitLab</li>
-          <li>Terminal</li>
-        </ul>
-      </div>
-    </div>
+  <section class="library-container">
+    <ul class="card">
```
Comment (ShoiraTa) [inline index.html]: - [ ] It looks like recent changes after after reviewer's comments have affected the design 😢 , please check: 

Before  | After
------------- | -------------
image | image

More precisely, listed items are going out of container, and the font color is different:

image

Please return previous design it was awesome 🥺

File: index.html [hunk 3]
```diff
@@ -16,10 +16,15 @@
       <nav class="nav">
         <a href="">Welcome</a>
         <i class="fas fa-bars"></i>
```
Comment (ShoiraTa) [inline index.html]: - [ ] [OPTIONAL]: please notice that in Figma there is no logo on the left side for desktop views 😯 : 

Yours | Figma
---------|---------
image  | image

Please consider deleting logo from desktop view.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @denscholar,

Your design is really fantastic 😁 😁 . Kudos for animation 👏🏻 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 


## Optional suggestions

_Check the comments under the review._ 

***

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (denscholar) [reply index.html]: I understand your point. The text and colour didn't align with the Figma design which is quite understandable and I will work on it. For the overlapping, I am sorry, but you obviously did not view the page using your developer's tool which is a more accurate/recommended way to ascertain if the project meets the requirement or not. Besides, we are designing for desktop and not for laptop. See the page when viewed using the developer's tool which is qy=uite different from what you attached here. Thank you.

page

Comment (ShoiraTa) [reply index.html]: @denscholar to be honest I am a little bit offended by your comment, but that is ok. I was checking everything with developer tools, and of course I know that it is more accurate/recommended  😉 .  Here is the screen record: 

https://user-images.githubusercontent.com/77038610/144698651-e2ff2ead-1d06-4632-91a3-4cacd504ef73.mp4

As a developer you should consider different screen sizes, for example mine is more than`1440px` and I have this: 

image

Comment (denscholar) [reply index.html]: I am very sorry if you feel offended. I am currently working on the feedback.

**Review by Cathella — APPROVED** [review]
## STATUS: APPROVED 🟢

### Hi team 👋🏻, 

You have done a great job working on this project 💪🏻, worked on changes requested by reviewers until your project is nearly perfect! Kudos kudos to you 🎉🎉.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes

- [ ] On the desktop view, the contact form has a field for first name and last name. However this is not seen in your design. It would be nice if you added it. See the comparisons below 👇🏻 

| Your design | Figma design |
| --- | --- |
| my4m | myfg |


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**